types/module/module.go

```diff
@@ -701,16 +701,17 @@ func (m Manager) RunMigrations(ctx context.Context, cfg Configurator, fromVM Ver
 
 // RunMigrationBeginBlock performs begin block functionality for upgrade module.
 // It takes the current context as a parameter and returns a boolean value
-// indicating whether the migration was successfully executed or not.
-func (m *Manager) RunMigrationBeginBlock(ctx sdk.Context) bool {
+// indicating whether the migration was executed or not and an error if fails.
+func (m *Manager) RunMigrationBeginBlock(ctx sdk.Context) (bool, error) {
```

Comment:
Returning an error makes sense, but having a book that is "err == nil" seems unnecessary. What is the reason for keeping it?